env0/resource_custom_flow.go

```diff
@@ -0,0 +1,211 @@
+package env0
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCustomFlow() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCustomFlowCreate,
+		ReadContext:   resourceCustomFlowRead,
+		UpdateContext: resourceCustomFlowUpdate,
+		DeleteContext: resourceCustomFlowDelete,
+
+		Importer: &schema.ResourceImporter{StateContext: resourceCustomFlowImport},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Description: "id of the custom flow",
+				Computed:    true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Description: "name for the custom flow",
+				Required:    true,
+			},
+			"repository": {
+				Type:        schema.TypeString,
+				Description: "repository url for the custom flow source code",
+				Required:    true,
+			},
+			"path": {
+				Type:        schema.TypeString,
```

Comment:
maybe we should indicate in the `path` description that it should be the full  path including .yaml/.yml

Reply:
@yarivg - good point. I will fix.

Reply:
added.